commit 1749d4af79d79c6011e9d195ebdbf36a0e8a9752 (refs/changes/60/960/2)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2016-11-29T10:09:06-08:00 (2 years, 10 months ago)
    
    DAPI-329 DAPI should use node v4.x
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/lib/allocator.js b/lib/allocator.js
index 51c1cfa..f18994f 100644
--- a/lib/allocator.js
+++ b/lib/allocator.js
@@ -50,7 +50,6 @@
  */
 
 var mod_fs = require('fs');
-var create_uuid = require('node-uuid');
 
 var ALGORITHMS_PATH = __dirname + '/algorithms/';
 
diff --git a/package.json b/package.json
index 2170ddf..0bd8f6a 100644
--- a/package.json
+++ b/package.json
@@ -1,15 +1,15 @@
 {
   "name": "dapi",
   "description": "SmartDataCenter Designation API",
-  "version": "7.0.0",
+  "version": "7.1.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
     "assert-plus": "~1.0.0",
-    "bunyan": "~1.3.3",
-    "node-uuid": "~1.4.0",
+    "bunyan": "1.8.5",
+    "libuuid": "0.2.1",
     "joyent-schemas": "git://github.com/joyent/schemas.git#c661af34f9fa839aada1ff5c215067d9c86cbe89",
-    "jsprim": "~0.6.1"
+    "jsprim": "1.3.1"
   },
   "devDependencies": {
     "tape": "4.0.0"
diff --git a/test/algorithms/hard-filter-locality-hints.test.js b/test/algorithms/hard-filter-locality-hints.test.js
index 615e1e5..a4df3db 100644
--- a/test/algorithms/hard-filter-locality-hints.test.js
+++ b/test/algorithms/hard-filter-locality-hints.test.js
@@ -8,7 +8,7 @@
  * Copyright 2016 Joyent, Inc.
  */
 
-var genUuid = require('node-uuid');
+var genUuid = require('libuuid').create;
 var test = require('tape');
 
 var filter = require('../../lib/algorithms/hard-filter-locality-hints.js');
diff --git a/test/algorithms/hard-filter-vm-count.test.js b/test/algorithms/hard-filter-vm-count.test.js
index fb3c75c..c49ccb6 100644
--- a/test/algorithms/hard-filter-vm-count.test.js
+++ b/test/algorithms/hard-filter-vm-count.test.js
@@ -9,7 +9,7 @@
  */
 
 var test = require('tape');
-var genUuid = require('node-uuid');
+var genUuid = require('libuuid').create;
 var filter = require('../../lib/algorithms/hard-filter-vm-count.js');
 
 
diff --git a/test/algorithms/soft-filter-locality-hints.test.js b/test/algorithms/soft-filter-locality-hints.test.js
index 8d629cb..3059430 100644
--- a/test/algorithms/soft-filter-locality-hints.test.js
+++ b/test/algorithms/soft-filter-locality-hints.test.js
@@ -8,7 +8,7 @@
  * Copyright 2016 Joyent, Inc.
  */
 
-var genUuid = require('node-uuid');
+var genUuid = require('libuuid').create;
 var test = require('tape');
 
 var filter = require('../../lib/algorithms/soft-filter-locality-hints.js');
